lms/djangoapps/coursewarehistoryextended/migrations/0001_initial.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+import django.db.models.deletion
+import coursewarehistoryextended.fields
+
+def bump_pk_start(apps, schema_editor):
+    if not schema_editor.connection.alias == 'student_module_history':
+        return
+    StudentModuleHistory = apps.get_model("courseware", "StudentModuleHistory")
+    biggest_id = StudentModuleHistory.objects.all().order_by('-id').first()
```

Comment:
In loadtesting it was invisible (took longer to wait for ./manage.py lms migrate to calculate state)
I just pulled the query out of the django shell and it finishes on the read replica in 0.01s - Here is the query for ref:

`SELECT `courseware_studentmodulehistory`.`id`, `courseware_studentmodulehistory`.`student_module_id`, `courseware_studentmodulehistory`.`version`, `courseware_studentmodulehistory`.`created`, `courseware_studentmodulehistory`.`state`, `courseware_studentmodulehistory`.`grade`, `courseware_studentmodulehistory`.`max_grade` FROM `courseware_studentmodulehistory` ORDER BY `courseware_studentmodulehistory`.`id` DESC limit 1;`